File: homework_1/2.min_of_three/min_of_three.cpp
#include <iostream>
using namespace std;

int main(){
    int x, y, z, min;
    cin >> x >> y >> z;

    min = x;
    if(y < min)
        min = y;
    if(z < min)
        min = z;
    
    cout << min << endl;

    return 0;
}

File: homework_1/2.min_of_three/min_of_three.java
import java.util.Scanner;

class min_of_three {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int x = input.nextInt();
        int y = input.nextInt();
        int z = input.nextInt();

        int min = x;
        if(y < min)
            min = y;
        if(z < min)
            min = z;

        System.out.println(min);

        input.close();
    }
}